server/portal/settings/settings_secret.example.py

```diff
@@ -253,30 +261,13 @@
     }
 }
 
-
-########################
-# DJANGO CMS SETTINGS
-########################
-
-# CMS Site (allows for multiple sites on a single CMS)
-_SITE_ID = 1
-
-_HAYSTACK_CONNECTIONS = {
-    'default': {
-        'ENGINE': 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
-        'URL': 'core_portal_elasticsearch:9200/',
-        'INDEX_NAME': 'cms',
-    }
-}
-
-_COMMUNITY_INDEX_SCHEDULE = None
-
 ########################
 # Custom Portal Template Assets
 # Asset path root is static files output dir.
 # {% static %} won't work in conjunction with {{ VARIABLE }} so use full paths.
 ########################
 
+# Default Art.
```

Comment:
__Question__

One interpretation of Stache's comments here suggest that an empty value is valid. Is it?

```
_PORTAL_ICON_FILENAME=''                 # Empty string yields NO icon.
```

Reply:
Seems that empty string will also result in `img/favicon.ico`

Reply:
Then the secret/setting looks good as is.

_That URL seems wrong. I think we should not support an empty value. I'd even advocate removing the else clause in `base.html` which reads the secret… but not in this PR. [FP-263](https://jira.tacc.utexas.edu/browse/FP-263) exists to revisit the favicon._

<details>
<summary>thoughts before I remembered <a href="https://jira.tacc.utexas.edu/browse/FP-263">FP-263</a></summary>

Hm, `/server/portal/templates/base.html` has `<link rel="icon" href="{% static 'img/favicon.ico' %}">` in its else clause, which should render as `static/img/favicon.ico`.

</details>

Reply:
Edited previous comment. (I hadn't noticed the comment was added back in from Stache.)